external/fv3fit/tests/conftest.py

```diff
@@ -13,3 +13,9 @@ def state(tmp_path_factory):
     lpath = tmp_path_factory.getbasetemp() / "input_data.nc"
     lpath.write_bytes(r.content)
     return xr.open_dataset(str(lpath))
+
+
+def pytest_configure(config):
```

Comment:
I would add this to `pytest.ini` instead since we have a couple over there already.

https://github.com/ai2cm/fv3net/blob/d4924b40618a363a5ad27a98b6fb410f541d7a48/pytest.ini#L18

Reply:
Thanks, I wasn't sure we had a pytest.ini.